code/game/jobs/job/engineering.dm

```diff
@@ -59,7 +59,7 @@
 	supervisors = "the chief engineer"
 	department_head = list("Chief Engineer")
 	selection_color = "#fff5cc"
-	department_access = list(ACCESS_ATMOSPHERICS, ACCESS_CE, ACCESS_CONSTRUCTION, ACCESS_EMERGENCY_STORAGE, ACCESS_ENGINE, ACCESS_ENGINE_EQUIP, ACCESS_EVA, ACCESS_EXTERNAL_AIRLOCKS, ACCESS_HEADS, ACCESS_KEYCARD_AUTH, ACCESS_MAINT_TUNNELS, ACCESS_MECHANIC, ACCESS_MINERAL_STOREROOM, ACCESS_MINISAT, ACCESS_RC_ANNOUNCE, ACCESS_SEC_DOORS, ACCESS_TCOMSAT, ACCESS_TECH_STORAGE, ACCESS_TELEPORTER)
+	department_access = list(ACCESS_ATMOSPHERICS, ACCESS_CE, ACCESS_CONSTRUCTION, ACCESS_EMERGENCY_STORAGE, ACCESS_ENGINE, ACCESS_ENGINE_EQUIP, ACCESS_EVA, ACCESS_EXTERNAL_AIRLOCKS, ACCESS_MAINT_TUNNELS, ACCESS_MECHANIC, ACCESS_MINERAL_STOREROOM, ACCESS_MINISAT, ACCESS_RC_ANNOUNCE, ACCESS_TCOMSAT, ACCESS_TECH_STORAGE, ACCESS_TELEPORTER)
```

Comment:
I do no think any non command should have access to the office of a head, even on lowpop. so this comment goes for most of the departments like for example a doctor with CMO office access.(Which is what i am guessing ACCESSS_CMO is. In my opinion that is.